src/common/transformations/include/ov_ops/rotary_positional_embeddings.hpp

```diff
@@ -27,6 +27,7 @@ class TRANSFORMATIONS_API RoPE : public Op {
         bool is_interleaved = false;   // interleaved mode, implies trans0213 happens after RoPE
         size_t rotary_ndims = 0;       // dimensions to be embedded (d in the description)
         bool is_chatglm = false;       // chatglm is special which overrides other setting
+        bool is_chatglm4 = false;      // chatglm4 is special which overrides other setting
```

Comment:
Could we use a transpose order to describe the input order/output order instead of using model-specific config like `config.input_trans0213`

Reply:
Modify config to support_2d_rope and update comments. please review it again.